R/csv.R

```diff
@@ -719,7 +719,7 @@ read_csv_metadata <- function(csv_file) {
         dense_inv_metric <- TRUE
       } else if (inv_metric_next) {
         inv_metric_split <- strsplit(gsub("# ", "", line), ",")
-        numeric_inv_metric_split <- rapply(inv_metric_split, as.numeric)
+        numeric_inv_metric_split <- suppressWarnings(rapply(inv_metric_split, as.numeric))
```

Comment:
If the model has no parameters but `fixed_param` is FALSE this would produce warnings. Simplest thing to do is just suppress.